src/instruments/src/EFB/Localization/eu.json

```diff
@@ -478,6 +478,7 @@
       "PauseAtTod": "T/D Pause",
       "PauseAtTodDistance": "T/D Pause Distance (nm)",
       "SeparateTillerFromRudderInputs": "Separate Tiller from Rudder Inputs",
+      "SyncEfis": "Sync EFIS controls from Captain to FO",
```

Comment:
no required - all  languages will be updated by the build process
you can do this to get all languages with the new key:
`npm run build:efb-translation local`
Then you can commit the language files with your new key as well.